tests/template_tests/test_autoreloader.py

```diff
@@ -56,6 +56,17 @@ def test_non_template_changed_in_template_directory(self, mock_reset):
         self.assertIsNone(autoreload.template_changed(None, Path(__file__)))
         mock_reset.assert_not_called()
 
+    @mock.patch("django.forms.renderers.get_default_renderer")
+    def test_form_template_reset(self, mock_reset):
+        template_path = Path(__file__).parent / "templates" / "index.html"
+        self.assertTrue(autoreload.template_changed(None, template_path))
```

Comment:
```suggestion
        self.assertIs(autoreload.template_changed(None, template_path), True)
```
This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).